Gem/Code/Source/Automation/ScriptReporter.cpp

```diff
@@ -1081,4 +1091,56 @@ namespace AtomSampleViewer
+            const AZStd::string screenshotWarningsLogLine = AZStd::string::format("Screenshot warnings: %u \n", scriptReport.m_screenshotWarningCount);
+            const AZStd::string failedScreenshotsLogLine = "\nFailed screenshot test info below.\n";     
+
+            const AZStd::string projectPath = AZ::Utils::GetProjectPath();
+            AZStd::chrono::system_clock::time_point now = AZStd::chrono::system_clock::now();
+            float timeFloat = AZStd::chrono::duration<float>(now.time_since_epoch()).count();
+            AZStd::string timeString = AZStd::string::format("%.4f", timeFloat);
+            AZStd::string exportFileName = AZStd::string::format("exportedTestResults_%s.txt", timeString.c_str());
+            AZStd::string exportTestResultsFolder;
+            AzFramework::StringFunc::Path::Join(projectPath.c_str(), "TestResults/", exportTestResultsFolder);
+
+            auto io = AZ::IO::LocalFileIO::GetInstance();
+            io->CreatePath(exportTestResultsFolder.c_str());
+            AZStd::string exportFile;
+            AzFramework::StringFunc::Path::Join(exportTestResultsFolder.c_str(), exportFileName.c_str(), exportFile);
+            AZ::IO::HandleType logHandle;
+
+        if (io->Open(exportFile.c_str(), AZ::IO::OpenMode::ModeWrite, logHandle))
+        {
+            io->Write(logHandle, assertLogLine.c_str(), assertLogLine.size());
+            io->Write(logHandle, errorsLogLine.c_str(), errorsLogLine.size());
+            io->Write(logHandle, warningsLogLine.c_str(), warningsLogLine.size());
+            io->Write(logHandle, screenshotErrorsLogLine.c_str(), screenshotErrorsLogLine.size());
+            io->Write(logHandle, screenshotWarningsLogLine.c_str(), screenshotWarningsLogLine.size());
+            io->Write(logHandle, failedScreenshotsLogLine.c_str(), failedScreenshotsLogLine.size());
+
+            for (const ScreenshotTestInfo& screenshotTest : scriptReport.m_screenshotTests)
+            {
+                const AZStd::string baselineImageLogLine = AZStd::string::format("Golden (baseline) image: %s \n", screenshotTest.m_localBaselineScreenshotFilePath.c_str());
+                const AZStd::string screenshotImageLogLine = AZStd::string::format("Test screenshot: %s \n", screenshotTest.m_screenshotFilePath.c_str());
+                const AZStd::string toleranceLevelLogLine = AZStd::string::format("Tolerance level: %s \n", screenshotTest.m_toleranceLevel.ToString().c_str());
+                const AZStd::string officialComparisonLogLine = AZStd::string::format("Image comparison result: %s \n", screenshotTest.m_officialComparisonResult.GetSummaryString().c_str());
+
+                io->Write(logHandle, baselineImageLogLine.c_str(), baselineImageLogLine.size());
+                io->Write(logHandle, screenshotImageLogLine.c_str(), screenshotImageLogLine.size());
```

Comment:
Are these going to be useful to see when an external contributor pastes their exportResults.txt file?

Reply:
Good point, I removed them since they can just look in the standalone GUI for the path to the failed screenshots